code/modules/busy_space/organizations.dm

```diff
@@ -435,9 +435,11 @@
 	destination_names = list(
 						"Venus",
 						"Earth",
+						"Mercury",
 						"Luna",
 						"Mars",
-						"Titan"
+						"Titan",
+						"Europa"
```

Comment:
vorestation edits, or, put this in another file and just override that list

Reply:
I think I added this to the wrong file. I overrode it in busy_space_virgo, or so I thought.